### PR TITLE
Gabriel/cs/drag drop tabs

### PR DESCRIPTION
**Pull request by gtangelo**
Added drag and drop functionality to the course tabs in the course selector.

Note: When adding this functionality, I removed the use of antd's tab classes/components. So the styling and UI functionality of the drag and drop tabs may have some slight variations to the one from the antd theme.

https://user-images.githubusercontent.com/65492346/162558010-b2d2f739-107d-4073-91ed-4f4a912fddfb.mp4

### CODE REVIEW

File: frontend/src/pages/CourseSelector/courseTabs/CourseTab.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const [scrolledTo, setScrolledTo] = useState(false);
+  const dispatch = useDispatch();
+  const ref = useRef(null);
+  const tabInView = useIntersectionObserver(ref);
+  const { active } = useSelector((state) => state.tabs);
+
+  const getDraggableStyle = (style, snapshot) => {
+    // lock x axis when dragging
+    if (style.transform) {
+      const axisLockX = `${style.transform.split(",").shift()}, 0px)`;
```
Comment (Peedee2002) [inline frontend/src/pages/CourseSelector/courseTabs/CourseTab.jsx]: this variable is only used once, just put your style in the transform dict

**Review by Peedee2002 — APPROVED** [review]
nice! and thanks for the video demo. 1 small change and we can merge this.

**Comment by gtangelo** [comment]
@Peedee2002 I added some changes again, mainly refactoring the navbar and course selector page css stylesheets to use css variables. This mainly relates to fixing the height of the navbar and course selector components as there was some issues after merging in the latest commit from dev. Let me know what you think on the changes and we can merge this in to dev! 😄

**Review by Peedee2002 — APPROVED** [review]
lgtm

**Pull request merged**